test/controllers/staff_dashboard_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class StaffDashboardControllerTest < ActionController::TestCase
+=begin
```

Comment:
I get assert_response :found as true for the non-admin fixture (aka bob). As in it redirects to /staff_dashboard instead of root_path